hashicorp/boundary-ui
# Target aliases list view

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-12955)

## Description

This PR adds a list of associated aliases in the side bar

## Screenshots (if appropriate):

**When there is no alias associated:**
![without alias association](https://github.com/hashicorp/boundary-ui/assets/15043878/276a0908-4a33-44e6-af36-f521ddf61554)
**list view:**
![alias list view](https://github.com/hashicorp/boundary-ui/assets/15043878/35bd3011-557c-4125-b785-6b6e3d7811a0)
**flyout:**
![flyoyt](https://github.com/hashicorp/boundary-ui/assets/15043878/5590e8d6-7ae8-4a95-8a31-2166a9daf2b4)


https://github.com/hashicorp/boundary-ui/assets/15043878/a93f6c54-14ee-4e2d-9eb5-96d9c8455b40




## How to Test

1. checkout latest from the boundary main and run the controller
2. In the UI, create a TCP target with 0 or more aliases 
3. After successful creation, you should be able to view them on the right sidebar
4. If there are no aliases, you should see an option to add a new one. Currently it is just a button and does not have a route attached. 




:technologist: [Admin preview](PATH_TO_FEATURE)

:desktop_computer: [Desktop preview](PATH_TO_FEATURE)

## Checklist:
